docs/pages/upgrading/agent-managed-updates/bootstrap-ec2-agent.mdx

```diff
@@ -0,0 +1,333 @@
+
+Please note:
+
+  - This invitation token will expire in 60 minutes
+  - example.teleport.sh:3025 must be reachable from the new node
+```
+
+After executing this command, we should receive generated join token name that will be used in the following steps
+of the cloud-init script to generate the Teleport configuration file.
+
+If you need something that lasts longer than a few hours, prefer to create a token resource with a concrete
+`metadata.expires` timestamp instead of cranking `--ttl` to very large values.
+
+Example (set an appropriate, bounded expiry):
+
+```yaml
+# token-long-lived.yaml
+kind: token
+version: v2
+metadata:
+  name: node-join-end-of-year
+  expires: 2025-12-31T23:59:00Z
+spec:
+  join_method: token
+  roles: [Node]
+```
+
+```code
+$ tctl create -f token-long-lived.yaml
+provision_token "node-join-end-of-year" has been created
+```
+
+With the token created by the resource configuration, `node-join-end-of-year` is the secret join token
+name that must be used in the next steps with the cloud-init script, instead of the auto-generated token
+created by the `tctl tokens add` command (<Var name="place.your.join.token-xxxx" />).
+
+<Admonition type="note">
+    To create a delegated token that allows a cluster to be joined via IAM, read the full guide and examples here:
+    [Joining Services via AWS IAM Role](../../enroll-resources/agents/aws-iam.mdx).
+</Admonition>
+
+## Create an EC2 Instance
+
+### Via the AWS web console
+
+Follow the procedure for [launching an instance](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-launch-instance-wizard.html)
+and [providing user data](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/user-data.html),
+which allows you to supply cloud-init directives at launch - whether by entering them in the User data field of the
+AWS Management Console's Advanced details section.
+
+In the example below, the directives create and configure a Teleport node on Amazon Linux 2.
+The script installs Teleport and the v2 updater and enables SSH service only.
+The `#cloud-config` line at the top is required in order to identify the commands as cloud-init directives.
+
+User-data example with a previously created join token:
+
+```yaml
+#cloud-config
+packages:
+- curl
+
+runcmd:
+  # Install Teleport using your cluster's script and define the group by query parameter.
+  # Script persists your updater flags (group, proxy, etc.)
+  - curl "https://<Var name="example.teleport.sh:443" />/scripts/install.sh?group=<Var name="development" />" | sudo bash
+
+  # Write agent config (join token) before starting the service
+  - |
+    cat >/etc/teleport.yaml <<EOF
+    version: v3
+    teleport:
+      join_params:
+        method: token
+        token_name: <Var name="place.your.join.token-xxxx" />
+      proxy_server: <Var name="example.teleport.sh:443" />
+    ssh_service:
+      enabled: true
+    auth_service:
+      enabled: false
+    proxy_service:
+      enabled: false
+    EOF
```

Comment:
It might be better to use `teleport configure` instead of crafting the configuration manually. This way we can change the defaults in a future teleport version and users won't have to change their init scripts.

```suggestion
  - teleport configure --roles node --proxy <Var name="example.teleport.sh:443" /> --join-method iam --token <Var name="place.your.join.token-xxxx" />
```